docs/api/navigation-history.md

```diff
@@ -0,0 +1,26 @@
+## Class: NavigationHistory
+
+> Handles session history, maintaining a list of navigation entries for backward and forward navigation.
+Process: [Main](../glossary.md#main-process)<br />
+_This class is not exported from the `'electron'` module. It is only available as a return value of other methods in the Electron API._
+
+### Instance Methods
+
+#### `navigationHistory.getActiveIndex()`
+
+Returns `Integer` - The index from which we would go back/forward or reload.
+
+#### `navigationHistory.getEntryAtIndex(index)`
+
+* `index` Integer
+
+Returns `Object`:
+
+* `url` string - The url of the navigation entry at the given index.
+* `title` string - The app title set by the page to be displayed on the tab.
```

Comment:
I think "the tab" might be the wrong abstraction for Electron since we don't have tabs implemented by default. I think "page title" is just good enough as it's a standard web concept (e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title).